org2jekyll.el

```diff
@@ -298,7 +338,8 @@ Return DEFAULT-VALUE if not found."
 (defvar org2jekyll-header-metadata nil
   "The needed headers for org buffer for org2jekyll to work.")
 
-(setq org2jekyll-header-metadata '(("title"       . 'mandatory)
+(setq org2jekyll-header-metadata '(("type"        . 'mandatory)
```

Comment:
I think it needs to be mandatory.
The idea is when creating a new draft, users choose what type/collection an article belongs to. They are not prompted to choose layout anymore.
I think layouts for a type of articles can be configured in org publish project and filled in automatically as usually articles in a type share the same layout.